app/app/settings.py

```diff
@@ -123,6 +123,13 @@
     if path.isdir(static_path)
 ]
 
+DEFAULT_FILE_STORAGE = 'storages.backends.azure_storage.AzureStorage'
```

Comment:
Django Storages is really neat. I've been meaning to add this for a while but haven't gotten around to it yet, so thanks for doing this!

Currently doccano uses [Apache libcloud](https://github.com/apache/libcloud) as a cloud storage abstraction layer to enable data import from Azure Storage, AWS S3, etc. via [Django Cloud Browser](https://github.com/ryan-roemer/django-cloud-browser):

https://github.com/CatalystCode/doccano/blob/5942bbf03cfaad9f2b3152618ab01c7507979be4/app/app/settings.py#L65-L67

As such, I would recommend to have this change mirror the same capability to support many cloud storage providers and use the [libcloud plugin for Django Storages](https://django-storages.readthedocs.io/en/latest/backends/apache_libcloud.html) instead of the direct Azure Storage one.